plugins/depthCamera/src/DepthCameraDriver.cpp

```diff
@@ -97,7 +104,7 @@ bool GazeboYarpDepthCameraDriver::open(yarp::os::Searchable &config)
 //     memset(m_RGBPointCloud_Buffer, 0x00, m_RGBPointCloud_BufferSize);
```

Comment:
The `m_RGBPointCloud_Buffer` seems not to be used at all, but is anyway deleted in https://github.com/robotology/gazebo-yarp-plugins/pull/232/files#diff-eff3bb8f5948742479396b9546d89fecR159 . Can we remove it?